src/App.js

```diff
@@ -18,8 +19,9 @@ function App() {
         <Route path="/about" element={<About />} />
         <Route path="/contact" element={<Contact />} />
         <Route path="/projects" element={<Projects />} />
-        <Route path="/profile/:username" element={<Profile />} />
-        <Route path="*" element={<ErrorPage />} /> 
+        <Route path="/profile" element={<Profile />} />
+        <Route path="/profile/:username" element={<ProfileDetails />} />
```

Comment:
Also changing path /profile/:username to /profile/:userid or /profile/:id could be a good idea